server/datastore/mysql/migrations/tables/20240701113709_VPPDBUpdates.go

```diff
@@ -0,0 +1,85 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240701113709, Down_20240701113709)
+}
+
+func Up_20240701113709(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+-- This table is the VPP equivalent of the host_software_installs table.
+-- It tracks the installation of VPP software on particular hosts.
+CREATE TABLE host_vpp_software_installs (
+	id int(10) unsigned NOT NULL AUTO_INCREMENT,
+	host_id INT(10) UNSIGNED NOT NULL,
+	
+	-- This is the adam_id of the VPP software that's being installed
+	adam_id VARCHAR(16) NOT NULL,
+
+	-- This is the UUID of the MDM command issued to install the software
+	command_uuid VARCHAR(127),
+	user_id INT(10) UNSIGNED NULL,
+
+	-- This indicates whether or not this was a self-service install
+	self_service TINYINT(1) NOT NULL DEFAULT FALSE,
+	
+	-- This is an ID for the event of "associating" the software with a host. 
+	-- This value comes from the "eventId" field in the response here:
+	-- https://developer.apple.com/documentation/devicemanagement/associate_assets
+	associated_event_id VARCHAR(36),
+
+	created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	updated_at TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+	PRIMARY KEY(id),
+	FOREIGN KEY (user_id) REFERENCES users (id) ON DELETE SET NULL
```

Comment:
Should probably be an FK for the `adam_id` too? And should we think about how to handle its deletion (CASCADE or SET NULL? Will we need the row in this table again after the app gets deleted? It depends a bit on how activities get stored - if all info is in the activity itself, we can safely delete the install request when the app gets deleted. If not, things get a bit more complicated...

Reply:
@mna looks like the activities will have the app name, adamID, and command UUID from command issued to install. I think those can all be stored with the activity, because none of those values are really expected to change so no need to pull them from here. So we can delete rows from here. Does that make sense?

Reply:
Yes! That's great news and makes sense to me!